### PR TITLE
tns commands will print error sources with line and column information pointing to .ts files

### PR DESCRIPTION
**Pull request by PanayotCankov**

### CODE REVIEW

File: bin/tns [hunk 1]
```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env node
 
 "use strict";
+
+require("source-map-support/register");
```
Comment (rosen-vladimirov) [inline bin/tns]: I believe this will fail if CLI is packed, as we do not have dependency to `source-map-support`

File: package.json [hunk 2]
```diff
@@ -105,7 +105,7 @@
     "istanbul": "0.4.5",
     "mocha": "3.1.2",
     "should": "7.0.2",
-    "source-map-support": "^0.4.14",
+    "source-map-support": "^0.4.18",
```
Comment (rosen-vladimirov) [inline package.json]: this should be strict version and maybe this should not be devDependency

Comment (PanayotCankov) [reply package.json]: So moving this to dependencies should also resolve the previous comment right?

**Comment by Plamen5kov** [comment]
Please add description to show why this is necessary.

**Comment by dtopuzov** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Comment by rosen-vladimirov** [comment]
I'm closing this PR as the proposed change will break CLI's execution when installed from npm. We do not distribute source-maps, so even if we move the source-map plugin to dependencies section, the execution will still fail at runtime.

**Pull request closed without merging**